src/Tile.php

```diff
@@ -21,7 +21,7 @@ public function jsonSerialize(): array
         return [
             "label" => $this->label,
             "count" => $this->count,
-            "description" => $this->description,
+            "description" => $this->description ?: null,
```

Comment:
`$this->description` is always a string. No need to return anything else here, I think.